### ISSUE TITLE
2 days remaining best use of time

### ISSUE THREAD

**Issue by Cleop** [dwyl/dwyl-site#320]
I have 2 days left to try and get the site live. There are several issues still open but some of them are large and some of them I require input from others or have questions around them.

In my opinion I would prioritise #309 before anything else. As for the others, I don't mind which I do but want to make sure that it's clear I can't do all of them and can the scope of any of them be reduced at all?

There are time estimates on the issues but I think they are inaccurate. It's been hard for me to estimate as I've never done some of these before and so don't know what I'm dealing with. It's also not clear to me which bits of work I should be doing and where I can ask for input from Mark or Harry ie. on reproducing font awesome icons.

#283 - tweaks to the team page to bring closer to the designs - blocker - can I use js or not to achieve this? #317 Is this extra work needed or are we happy as they are?
#222 - detailed google analytics - Harry's metrics are quite complex and require a data layer and js to create his custom queries. So again, #317 can I use js to achieve this? To create all of his queries would take around a couple of days. Daniel thinks each custom bullet might take around 2 hours. I'm not suggesting Harry's metrics aren't useful but the question is, are they the first ones we want  building? do we know why? do we need all of them?
#23 - I can look into image optimisation. I've already made the images smaller but lost the quality and so couldn't implement them. Nelson wants the pages to be 100kb on load, is this realistic? Google's homepage is 500kb on load for example. I can get rid of font awesome which will help but I need substitute icons. Should I be making these or should I ask Harry or someone else for them?
#220 - I can do this but again, won't be very fast. Is this a priority for us right now? Have we tested or thought about this quite complex way of generating fields? Do people like having specific fields to fill in or do they prefer a simple open text field? 
#218 - I started on this and got stuck as the documentation is very outdated and so hard to follow and it spans about 3 different google product. Can continue but won't be fast.
#309 - in my opinion this is a high priority as it currently doesn't work on the site. I created the issue so it's not got a priority label from Mark.

**Comment by iteles** [comment]
@Cleop Please do make a start on #309 as you suggest, I'll review the rest in @markwilliamfirth's absence

**Comment by iteles** [comment]
I really don't understand the use of priority labels or milestones in this repo so I'm struggling to put things in an orderly fashion. Therefore, here are the proposed priorities for _today_:
+ #309 
+ #283 timeboxed - could you please add a separate issue for adding JS to the team page and estimate It separately?
+ #222 - this is actually more important then #283 for me, but the team page needs to look acceptable and this doesn't need to be as complex. Will comment in the issue 👌🏻

**Comment by Cleop** [comment]
Closing as the time is now over.
